about.html

```diff
@@ -0,0 +1,94 @@
+                        <li class="list-items1 campaign"><a href="#">Cc Campaign</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <section class="about-main">
+            <div class="about-main-cont">
+                <div class="hamburger-btn">
+                    <div class="ham-line"></div>
+                    <div class="ham-line"></div>
+                    <div class="ham-line"></div>
+                </div>
+                <h1>THE PLATFORM NIGERIA 2021</h1>
+                <hr>
+                <p class="about-main-cont-p2">The Platform is a non-political and non-partisan national development fair which holds as a global media event on Workers’ Day (May 1) and Nigeria’s Independence Day (October 1). It hosts speakers from the highest levels of the nation’s leadership and leaders in global policy practice.</p>
+                <p class="about-main-cont-p3">Please contact us per Email for any further questions about The Platform 2021!</p>
+                <p class="about-main-cont-p4">theplatform2021@tpn.org></p>
+            </div>
+        </section>
+        <section class="about-sub">
+            <h3>The Platform Nigeria 2021 Logo</h3>
+            <hr>
+            <p>The logo of The Platform Nigeria was decided Lorem ipsum dolor sit amet consectetur adipisicing elit. Accusantium beatae quidem perferendis ducimus libero officiis, est consequatur! Ipsam, id suscipit animi soluta amet laudantium aliquid, porro consequuntur explicabo necessitatibus official</p>
+            <div class="about-sub-img">
+                <img src="images/The-Platform-Logo.png" alt="The plaform logo">
+            </div>
+        </section>
+        <section class="about-minor">
+            <h3 class="about-minor-h3">See the past The Plaform Nigeria Summits</h3>
+            <hr>
+            <p class="about-minor-p">take a look at the last two The Platform Nigeria which took place in Lagos and in Abuja</p>
+            <div class="about-flex">
+            <div class="about-cont1">
+                <div class="about-cont11">
+                    <h3 class="about-minor-h32">2018</h3>
+                    <p class="about-minor-p2">The Platform Nigeria 2018 Lagos</p>
+                </div>
+            </div>
+            <div class="about-cont2">
+                <div class="about-cont11">
+                    <h3 class="about-minor-h32">2019</h3>
+                    <p class="about-minor-p2">The Platform Nigeria 2019 Abuja</p>
+                </div>
+            </div>
+            </div>
+        </section>
+        <footer>
+            <div class="about-footer">
+                <img src="images/tcn-logo12.png" alt="TNC logo">
+                <div>
+                    <p class="about-footer-p1">Copyright &copy; 2021. All right reserved</p>
+                    <p class="about-footer-p2">Developed by Femi</p>
+                </div>
+            </div>
+        </footer>
```

Comment:
- [ ] Both pages should have the same navbar and footer but they are a little different between them, please make them be the same ones to comply with the project requirements.

Reply:
- [ ] Also, the footer should have a white background in the mobile view and a black one in the desktop view, please make this change to comply with the project requirements.